src/pykeen/hpo/hpo.py

```diff
@@ -103,13 +103,11 @@ def _update_stopper_callbacks(stopper_kwargs: Dict[str, Any], trial: Trial) -> N
         """Make a subclass of the EarlyStopper that reports to the trial."""
 
         def _continue_callback(early_stopper: EarlyStopper, result: Union[float, int]) -> None:
-            last_epoch = early_stopper.number_results * early_stopper.frequency
-            trial.report(result, step=last_epoch)
+            trial.report(result, step=early_stopper.current_epoch)
 
         def _stopped_callback(early_stopper: EarlyStopper, result: Union[float, int]) -> None:
-            current_epoch = (1 + early_stopper.number_results) * early_stopper.frequency
-            trial.set_user_attr(STOPPED_EPOCH_KEY, int(current_epoch))
-            trial.report(result, current_epoch)  # don't include a step because it's over
+            trial.set_user_attr(STOPPED_EPOCH_KEY, early_stopper.current_epoch)
+            trial.report(result, early_stopper.current_epoch)  # don't include a step because it's over
```

Comment:
The comment 

```python
 # don't include a step because it's over
```

is outdated.

Reply:
Done in cc218fd538efa200d7b8984c9e8f08ec1b1efe25.